Packs/DragosWorldview/Integrations/DragosWorldview/DragosWorldview.py

```diff
@@ -127,7 +128,7 @@ def fetch_incidents(client: Client, last_run: dict, first_fetch: str) -> Tuple[l
 
     max_time = last_fetch
 
-    api_query = "products?updated_after="
+    api_query = "products?released_after="
```

Comment:
I made this integration ~ 1 year ago, when I made it the only time based filtering was the released_after parameter. We have recently seen issues with pulling in old alerts because old alerts have been getting updated. They now have a released_after parameter which is more suitable for what we wanted in the first place.